src/sphinxexpr.cpp

```diff
@@ -2702,13 +2702,15 @@ class Expr_Case_c : public ISphStringExpr
 template<>
 void Expr_Case_c<true> :: DoCase ( char *pString ) const
 {
-    *pString = toupper(*pString);
+	// converts a letter into it's uppercase letter if exits.
```

Comment:
Just wondering: that is very obvious, calling library's tolower/toupper function is not a case where detailed commenting is necessary (it would so, for example, if you provide own non-obvious implementation with complex logic, not clear from first glance. But here it looks redundant).